### DISCUSSION TITLE
intel gpu not working

### DISCUSSION THREAD

**Discussion by Atishay9828** [lllyasviel/Fooocus#3732] [Q&A] [answered]
i tried using intel gp , i have 16gb of ram and 8.8 gb of shared memory with my intel arc gpu and npu so i should be able to run it but that;s not the case really would like it if fooocus team pushes out support for intel gpu;s too as they are also starting to gain numbers ....

**Comment by mashb1t** [comment, marked as answer]
please find community docs in https://github.com/lllyasviel/Fooocus/pull/2120 ([files](https://github.com/lllyasviel/Fooocus/pull/2120/files))
